docs/parse_argparse.py

```diff
@@ -310,21 +317,20 @@ def main():
         os.path.dirname(__file__), "source", "scripts")
     remove_existing_docs(rst_scripts_dirpath)
     remove_empty_dir()
-    pkg_scripts_fpath = os.path.join(os.path.dirname(
+    pkg_scripts_search_fpath = os.path.join(os.path.dirname(
         os.path.dirname(__file__)), "autometa", "**", "*.py")
-    for fpath in glob.glob(pkg_scripts_fpath, recursive=True):
+    for fpath in glob.glob(pkg_scripts_search_fpath, recursive=True):
```

Comment:
See comment for line 320 above

```suggestion
    for fpath in glob.glob(pkg_scripts_search_str, recursive=True):
```